lib/puppet/provider/grafana_datasource/grafana.rb

```diff
@@ -19,6 +19,10 @@ def grafana_api_path
     resource[:grafana_api_path]
   end
 
+  def grafana_api_path
```

Comment:
This doesn't look right. You're duplicating a method here.

Reply:
Thanks, that is the typical copy/paste error, when interrupted by phone calls. Resolved with https://github.com/voxpupuli/puppet-grafana/pull/300/commits/98e2735d82b8093fbe84f504e317508685430877